aea/protocols/generator/common.py

```diff
@@ -80,6 +80,10 @@
     "--quiet",
 ]
 
+PATH_TO_PROTOLINT_CONFIGURATION_FILE = os.path.join(
+    CURRENT_DIR, "..", "..", "..", "protolint.yaml"
+)
+
```

Comment:
Erm no, this won't work. The distribution does not have access to this.

Reply:
Since the yaml file is short you could store the whole spec as a string constant and then create a temp file for the cli call.

Reply:
Done